e2e/_suites/fleet/features/stand_alone_agent.feature

```diff
@@ -70,3 +70,219 @@ Examples: default
+  Given a "<image>" stand-alone agent is deployed
+    And the stand-alone agent is listed in Fleet as "online"
+  When the policy is updated to have "system/metrics" set to "memory"
+  Then "system/metrics" with "memory" metrics are present in the datastreams
+
+@ubi8
+@skip:arm64
+Examples: Ubi8
+| image |
+| ubi8  |
+
+@default
+Examples: default
+| image   |
+| default |
+
+@start-stand-alone-agent-with-network
+Scenario Outline: Adding the network System Integration to an stand-alone-agent
+  Given a "<image>" stand-alone agent is deployed
+    And the stand-alone agent is listed in Fleet as "online"
+  When the policy is updated to have "system/metrics" set to "network"
+  Then "system/metrics" with "network" metrics are present in the datastreams
+
+@ubi8
+@skip:arm64
+Examples: Ubi8
+| image |
+| ubi8  |
+
+@default
+Examples: default
+| image   |
+| default |
+
+@start-stand-alone-agent-with-process
+Scenario Outline: Adding the process System Integration to an stand-alone-agent
+  Given a "<image>" stand-alone agent is deployed
+    And the stand-alone agent is listed in Fleet as "online"
+  When the policy is updated to have "system/metrics" set to "process"
+  Then "system/metrics" with "process" metrics are present in the datastreams
+
+@ubi8
+@skip:arm64
+Examples: Ubi8
+| image |
+| ubi8  |
+
+@default
+Examples: default
+| image   |
+| default |
+
+@start-stand-alone-agent-with-socket_summary
+Scenario Outline: Adding the socket_summary System Integration to an stand-alone-agent
+  Given a "<image>" stand-alone agent is deployed
+    And the stand-alone agent is listed in Fleet as "online"
+  When the policy is updated to have "system/metrics" set to "socket_summary"
+  Then "system/metrics" with "socket_summary" metrics are present in the datastreams
+
+@ubi8
+@skip:arm64
+Examples: Ubi8
+| image |
+| ubi8  |
+
+@default
+Examples: default
+| image   |
+| default |
+
+@start-stand-alone-agent-with-uptime
+Scenario Outline: Adding the uptime System Integration to an stand-alone-agent
+  Given a "<image>" stand-alone agent is deployed
+    And the stand-alone agent is listed in Fleet as "online"
+  When the policy is updated to have "system/metrics" set to "uptime"
+  Then "system/metrics" with "uptime" metrics are present in the datastreams
+
+@ubi8
+@skip:arm64
+Examples: Ubi8
+| image |
+| ubi8  |
+
+@default
+Examples: default
+| image   |
+| default |
+
+#@start-stand-alone-agent-with-filesystem
+#Scenario Outline: Starting the <image> stand-alone-agent and check filesystem in datastreams
+#  Given a "<image>" stand-alone agent is deployed
+#  When the stand-alone agent is listed in Fleet as "online"
+#    And the policy is updated to have "system/metrics" set to "filesystem"
+#  Then "system/metrics" with "filesystem" metrics are present in the datastreams
+#
+#@ubi8
+#@skip:arm64
+#Examples: Ubi8
+#| image |
+#| ubi8  |
+#
+#@default
+#Examples: default
+#| image   |
+#| default |
```

Comment:
If we are not going to use it, I'd directly remove it. Wdyt?

Reply:
Done